File: C#/Com.Ericmas001.Game.Poker.Protocol.Commands/Game/GameEndedCommand.cs
﻿using System;
using System.Collections.Generic;
using System.Text;
using Com.Ericmas001;
using Com.Ericmas001.Net.Protocol.JSON;

namespace Com.Ericmas001.Game.Poker.Protocol.Commands.Game
{
    public class GameEndedCommand : AbstractJsonCommand
    {
        public static string COMMAND_NAME = "gameENDED";
    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/game/GameEndedCommand.java
package bluffinmuffin.protocol.commands.game;

import java.util.StringTokenizer;

import bluffinmuffin.protocol.commands.ICommand;


public class GameEndedCommand implements ICommand
{
    public static String COMMAND_NAME = "gameENDED";
    
    public GameEndedCommand(StringTokenizer argsToken)
    {
    }
    
    public GameEndedCommand()
    {
    }
    
    @Override
    public String encodeCommand()
    {
        final StringBuilder sb = new StringBuilder();
        sb.append(GameEndedCommand.COMMAND_NAME);
        return sb.toString();
    }
}
